packages/amplify-graphql-auth-transformer/src/graphql-auth-transformer.ts

```diff
@@ -209,10 +211,9 @@ Static group authorization should perform as expected.`,
     const modelDirective = parent.directives?.find(dir => dir.name.value === 'model');
     const typeName = parent.name.value;
     const fieldName = field.name.value;
-    const authDir = new DirectiveWrapper(directive);
-    const rules: AuthRule[] = authDir.getArguments<{ rules: Array<AuthRule> }>({ rules: [] }).rules;
-    ensureAuthRuleDefaults(rules);
-    validateFieldRules(rules, isParentTypeBuiltinType, modelDirective !== undefined, this.configuredAuthProviders, field.name.value);
+    const rules: AuthRule[] = getAuthDirectiveRules(new DirectiveWrapper(directive));
+    validateFieldRules(new DirectiveWrapper(directive), isParentTypeBuiltinType, modelDirective !== undefined, field.name.value);
```

Comment:
nit: can we pass in the authdirective rules here?
```suggestion
    validateFieldRules(rules, isParentTypeBuiltinType, modelDirective !== undefined, field.name.value);
```

Reply:
the reason for this is that the validation needs to happen on the directive as provided by the customer without any preprocessing done on it